front/src/NotificationWindow.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import styled from "styled-components";
+import faTimes from "@fortawesome/fontawesome-free-solid/faTimes";
+import FontAwesomeIcon from "@fortawesome/react-fontawesome";
+
+const Window = styled.div`
+  background-color: ${props => props.backgroundColor};
+  border-style: solid;
+  border-width: 0.4rem;
+  width: 90%;
+  position: relative;
+  margin: 0 auto;
+`;
+
+const Header = styled.h1`
+  text-align: center;
+  font-size: 5rem;
+`;
+
+const Text = styled.p`
+  padding-left: 10px;
+  padding-right: 10px;
+  text-align: center;
+  font-size: 8rem;
+`;
+
+const CloseButton = styled.button`
+  border: none;
+  background-color: transparent;
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 10rem;
+  height: 10rem;
+  font-size: 2rem;
+`;
+
+/**
+ * Creates a notification window
+ * Author: S.A. Tanja
+ *
+ * Props:
+ * - backgroundColor: sets the background color of the notification window
+ * - notificationType: sets the title of the notification window
+ * - onClick: on close function should be passed
+ * - message: the message of the notification window
+ * - children: possible extra elements that could be included (buttons, etc)
+ */
+const NotificationWindow = props => {
+  return (
+    <Window backgroundColor={props.backgroundColor}>
+      <Header>{props.notificationType}</Header>
```

Comment:
`notificationType` = title?

Reply:
Now `notificationType` is used for the background color, so it remains `notificationType`